modules/apps/frontend-js/frontend-js-web/src/main/resources/META-INF/resources/liferay/DynamicInlineScroll.es.js

```diff
@@ -72,7 +72,10 @@ class DynamicInlineScroll extends PortletBase {
 	addListItem_(listElement, pageIndex) {
 		const listItem = document.createElement('li');
 
-		listItem.innerHTML = `<a class="dropdown-item" href="${this.getHREF_(
+		listItem.innerHTML = `<a aria-label="${Liferay.Util.sub(
+			Liferay.Language.get('page-x'),
+			[pageIndex]
+		)}" class="dropdown-item" href="${this.getHREF_(
```

Comment:
Hi @oliv-yu , we're getting closer :sweat_smile: 

We need to remove the `<span>` below (lines 80-82), as that one was the culprit of the a11y issue. This way link would just have the page number with no hidden markup. You can take the `start.jsp` as reference of how it should look like.

Thank you!

Reply:
Oh my mistake! I updated the recent commit to remove the span as well then, let me know if that's better! 😅